lib/perl/Genome/Model/SomaticValidation/Command/AlignReads.pm

```diff
@@ -105,5 +109,66 @@ sub execute {
     return 1;
 }
 
+sub _find_existing_alignments {
```

Comment:
I changed this to return the alignments instead.  I'd like to avoid an innocuous boolean-sounding name since this has a lot of side-effects!